### PR TITLE
Delete log message that prints argument list

### LINKED ISSUE #8929: Incorrect log message
Log message has the ToString() result of the collection.

> Worker description arguments after formatting: System.Collections.Generic.List`1[System.String]

Source: Host.LanguageWorkerConfig
Host version: 4.24.1.0

### PR DESCRIPTION
**Pull request by shreyas-gopalakrishna**
### Delete log message that prints argument list

Argument list can be found when worker process is started.
image

resolves #8929

### Pull request checklist

* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [ ] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

**Review by kshyju — APPROVED** [review]

File: src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs [hunk 1]
```diff
@@ -218,7 +218,7 @@ internal void FormatArgumentsIfNeeded(ILogger logger)
                 }
             }
 
-            logger.LogDebug($"Worker description arguments after formatting: {Arguments}");
+            logger.LogDebug($"Worker description arguments after formatting: {string.Join(" ", Arguments)} ");
```
Comment (kshyju) [inline src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs]: Are we not logging the arguments somewhere later (when we start the language worker process)? Is that not enough? Would this information redundant in that case?

Comment (liliankasem) [reply src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs]: @shreyas-gopalakrishna

Comment (shreyas-gopalakrishna) [reply src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs]: Yes, Found the below log that gives us the arguments info. This log can be removed.
image

**Review by liliankasem — APPROVED** [review]

**Review by satvu — APPROVED** [review]

**Comment by liliankasem** [comment]
@shreyas-gopalakrishna any blockers on this PR? can you merge?

**Review by shreyas-gopalakrishna — COMMENTED** [review]
@liliankasem the merge option isn't enabled as pipeline is failing. I am not able to get a green pipeline after multiple runs

**Comment by liliankasem** [comment]
> @liliankasem the merge option isn't enabled as pipeline is failing. I am not able to get a green pipeline after multiple runs

What are the test failures? Did you look into what was failing?

Edit: I reran pipeline for this branch and everything is green: https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=112513&view=results

Was there a bad rebase at some point? The GH checks below are saying there are 2 failures for merge commit ([a81ea48](https://github.com/Azure/azure-functions-host/commit/a81ea4879a7481f2e7dd68b23be910476ffec7d8)) which doesn't exist any more

**Pull request merged**